### PR TITLE
show terminal find history placeholder on focus

### LINKED ISSUE #148135: Terminal: find widget placeholder should mention up / down arrows for history navigation
Testing #148060

Notice how the editor nicely advertises the up and down arrows for history navigation.
Terminal find widget should do the same since it supports history navigation

Screenshot 2022-04-26 at 13 26 44
Screenshot 2022-04-26 at 13 26 47

### PR DESCRIPTION
**Pull request by meganrogge**
This PR fixes #148135

### CODE REVIEW

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts [hunk 1]
```diff
@@ -80,9 +81,9 @@ export abstract class SimpleFindWidget extends Widget {
 			},
 			appendCaseSensitiveLabel: options.appendCaseSensitiveLabel && options.type === 'Terminal' ? this._getKeybinding(TerminalCommandId.ToggleFindCaseSensitive) : undefined,
 			appendRegexLabel: options.appendRegexLabel && options.type === 'Terminal' ? this._getKeybinding(TerminalCommandId.ToggleFindRegex) : undefined,
-			appendWholeWordsLabel: options.appendWholeWordsLabel && options.type === 'Terminal' ? this._getKeybinding(TerminalCommandId.ToggleFindWholeWord) : undefined
+			appendWholeWordsLabel: options.appendWholeWordsLabel && options.type === 'Terminal' ? this._getKeybinding(TerminalCommandId.ToggleFindWholeWord) : undefined,
+			showHistoryHint: () => showHistoryKeybindingHint(_keybindingService)
```
Comment (meganrogge) [inline src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts:85]: this is what others do

**Comment by meganrogge** [comment]
Looks sort of bad with the width we have
Screen Shot 2022-05-25 at 12 03 12 PM

**Review by Tyriar — DISMISSED** [review]
I guess we could increase the width for now and clean it up if the find widget goes to per terminal in https://github.com/microsoft/vscode/issues/145865?

**Comment by meganrogge** [comment]
Screen Shot 2022-05-25 at 12 38 15 PM

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css [hunk 2]
```diff
@@ -45,6 +46,7 @@
 
 .monaco-workbench .simple-find-part .monaco-findInput {
 	flex: 1;
+	min-width: 220px;
```
Comment (Tyriar) [inline src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css:49]: Should this use flex-grow instead so it bases the width off its container?

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css [hunk 3]
```diff
@@ -9,7 +9,7 @@
 	position: absolute;
 	top: 0;
 	right: 18px;
-	width: 220px;
+	min-width: 360px;
```
Comment (Tyriar) [inline src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css:12]: Changes to how the width work are necessary? More to verify if we change it up

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css (hunk 2 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css:49]: I tried that and could be doing something, but it didn't work

File: src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css (hunk 3 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css:12]: yeah looks like that wasn't necessary

**Review by Tyriar — APPROVED** [review]

**Pull request merged**